wbia/algo/hots/neighbor_index_cache.py

```diff
@@ -487,8 +487,7 @@ def request_memcached_wbia_nnindexer(
     if (
         not force_rebuild
         and use_memcache
-        and nnindex_cfgstr
-        in NEIGHBOR_CACHE.has_key()  # NOQA (has_key is for a lru cache)
```

Comment:
Looks like maybe `nnindex_cfgstr in NEIGHBOR_CACHE` was maybe wanted. So it was maybe Jason's intent to change the *behavior* to use `in` rather than the `has_key` method. There was a bit of a bleed over of behavior change in what would otherwise have been a formatting commit.

Reply:
This is correct, this must have been my typo.